en/flying/tunes.md

```diff
@@ -0,0 +1,156 @@
+# Tune Meanings (Pixhawk Series)
+
+[Pixhawk-series flight controllers](../flight_controller/pixhawk_series.md) use audible tones/tunes to indicate vehicle state and events (e.g. arming success and failure, low battery warnings).
+
+The current set of tones are listed/can be played below.
+
+Startup tone - played on completion of startup, before arming.
```

Comment:
So, when played? What does it mean - basically just a notification that we're now searching for satellites?